app/viewModels/P5/departure/DepartureStatusP5ViewModel.scala

```diff
@@ -331,11 +339,19 @@ object DepartureStatusP5ViewModel {
       )
   }
 
-  private def goodsBeingRecovered(): PartialFunction[DepartureMessage, DepartureStatusP5ViewModel] = {
+  private def goodsBeingRecovered(departureId: String,
+                                  localReferenceNumber: LocalReferenceNumber
+  ): PartialFunction[DepartureMessage, DepartureStatusP5ViewModel] = {
     case message if message.messageType == GoodsBeingRecovered =>
```

Comment:
I don't think this is related to your change, but we would get a match error here if it is the wrong messageType.